test/edu/csus/ecs/pc2/core/imports/clics/CLICSScoreboardTest.java

```diff
@@ -0,0 +1,179 @@
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.exception.IllegalContestState;
+import edu.csus.ecs.pc2.core.log.Log;
+import edu.csus.ecs.pc2.core.log.StaticLog;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.core.model.JSONObjectMapper;
+import edu.csus.ecs.pc2.core.model.SampleContest;
+import edu.csus.ecs.pc2.core.scoring.DefaultScoringAlgorithm;
+import edu.csus.ecs.pc2.core.scoring.NewScoringAlgorithm;
+import edu.csus.ecs.pc2.core.standings.ContestStandings;
+import edu.csus.ecs.pc2.core.standings.ScoreboardUtilites;
+import edu.csus.ecs.pc2.core.standings.json.TeamScoreRow;
+import edu.csus.ecs.pc2.core.util.AbstractTestCase;
+
+/**
+ * Unit test.
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public class CLICSScoreboardTest extends AbstractTestCase{
+
+    /**
+     * 
+     * @param logDirectoryName
+     * @param runsData
+     * @return
+     * @throws Exception 
+     */
+    private IInternalContest loadSampleRuns( String [] runsData, String logDirectoryName) throws Exception {
+        SampleContest sample = new SampleContest();
+        IInternalContest contest = sample.createStandardContest();
+
+        Log log = new Log(logDirectoryName, getName()+".log");
+        StaticLog.setLog(log);
+
+        addRuns(contest, runsData);
+        return contest;
+    }
+
+
+    String[] getRunsData(){
+
+        String [] runsData = {
+
+                "1,1,A,1,No",  //20
+                "2,1,A,3,Yes",  //3 (first yes counts Minutes only)
+                "3,1,A,5,No",  //20
+                "4,1,A,7,Yes",  //20  
+                "5,1,A,9,No",  //20
+
+                "6,1,B,11,No",  //20  (all runs count)
+                "7,1,B,13,No",  //20  (all runs count)
+
+                "8,2,A,30,Yes",  //30
+
+                "9,2,B,35,No",  //20 (all runs count)
+                "10,2,B,40,No",  //20 (all runs count)
+                "11,2,B,45,No",  //20 (all runs count)
+                "12,2,B,50,No",  //20 (all runs count)
+                "13,2,B,55,No",  //20 (all runs count)
+
+                "14,3,A,130,Yes",  //30
+                "15,3,B,30,Yes",  //30
+                "16,4,A,230,Yes",  //30
+                "17,5,A,30,Yes",  //30
+                "18,5,B,130,Yes",  //30
+
+        };
+
+        return runsData;
+    }
+
+
+    private ContestStandings createContestStandings(IInternalContest contest) throws JsonParseException, JsonMappingException, IOException, IllegalContestState {
+
+        NewScoringAlgorithm scoringAlgorithm = new NewScoringAlgorithm();
```

Comment:
I've commented on this elsewhere: it bothers me that tests are using a different scoring algorithm class than the one used when PC2 is executed via a distribution...

Reply:
It bothers me too and that is why it was proposed for a quarterly cleanup to avoid pushing fixes like this off.
Rather than someday.

Reply:
I get that it's the kind of thing that tends to get "put off", and I've filed a new issue (#846) as you suggested so that we have a single-point reference regarding the problem.  However, I don't understand why, if you agree that using a different class from `DefaultScoringAlgorithm` is bothersome, you decided to use that class in this new code?

Reply:
Because NewScoringAlgorithm is the desired/better Java SA.   DSA should be deprecated, primarily because NSA is a Java class with objects (OOPS) whereas DSA just returns a XML string.